swagger.yaml

```diff
@@ -5368,6 +5368,10 @@ definitions:
         type: integer
         format: int32
         description: Defines whether this is an IPv4 (4) or IPv6 route (6)
+      metric:
```

Comment:
changes api?

Reply:
Yes...
But since it's an additional field it shouldn't break anything

Reply:
Yep. It doesn't